### DISCUSSION TITLE
please explain how I can go about using hasura CLI to export metadata via Windows Active Directory Login?

### DISCUSSION THREAD

**Discussion by newemployee909090** [hasura/graphql-engine#9194] [Question]
Here is information about our technical environment:

-Hasura GraphQL Current server version: v2.6.2-pro.1

-hasura CLI version 2.15.0

-We log onto the Hasura GraphQL Web UI Console using our Windows Active Directory Login ( essentially Single-SignOn SSO) ( Therefore, we do Not have an admin secret )

However, the official Hasrua GraphQL Technical Tutorial Guide only gives examples showing the admin secret being supplied in the hasura CLI command line console arguments ( https://hasura.io/docs/latest/migrations-metadata-seeds/migrations-metadata-setup/ )

hasura init demo-project --endpoint https://docs-demo.hasura.app/ --admin-secret mySecret

Could someone please explain how I can go about using hasura CLI to export metadata via Windows Active Directory Login? (Please provide hasura CLI command line examples )?

**Comment by adas98012** [comment]
@newemployee909090 
 You would need admin-secret (you can set in environment variable). The same question was raised in a different forum, wanted to share 
 https://stackoverflow.com/questions/74365956/how-can-i-go-about-using-hasura-cli-to-export-metadata-via-windows-active-direct
